### PR TITLE
926 add  set static property command

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@RobertPoienar @robert-altom @gomboskriszta15 @ReichEdvin

File: Bindings~/java/src/main/java/com/alttester/Commands/AltSetStaticProperty.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.alttester.Commands;
+
+import com.alttester.IMessageHandler;
+import com.alttester.Commands.ObjectCommand.AltSetComponentPropertyParams;
+
+/**
+ * Get the value of a property from one of the component of the object.
```
Comment (ReichEdvin) [inline Bindings~/java/src/main/java/com/alttester/Commands/AltSetStaticProperty.java:7]: It sets the value

File: Assets/AltTester/AltDriver/AltDriver.cs [hunk 2]
```diff
@@ -182,6 +182,12 @@ public T GetStaticProperty<T>(string componentName, string propertyName, string
             return propertyValue;
         }
 
+        public void SetStaticProperty(string componentName, string propertyName, string assemblyName, object updatedProperty, int maxDepth = 2)
```
Comment (RobertPoienar) [inline Assets/AltTester/AltDriver/AltDriver.cs:185]: maxDepth is not needed

File: Bindings~/java/src/main/java/com/alttester/AltDriver.java [hunk 3]
```diff
@@ -650,6 +651,19 @@ public <T> T getStaticProperty(AltGetComponentPropertyParams parameters, Class<T
         return response;
     }
 
+     /**
+     * Sets the value of the static field or property
+     *
+     * @param parameters - String componentName* , String propertyName* , String
```
Comment (RobertPoienar) [inline Bindings~/java/src/main/java/com/alttester/AltDriver.java:657]: Don't have/need maxDepth

**Review by RobertPoienar — CHANGES_REQUESTED** [review]
Added some comments, also you need to add this command to documentation

**Review by ReichEdvin — APPROVED** [review]
lgtm

**Review by RobertPoienar — APPROVED** [review]
LGTM

**Pull request merged**